javascript/react-js/hooks.md

```diff
@@ -156,6 +156,8 @@ If you write a return statement like the above in a useEffect, it will do the sa
 
 [Here](https://www.ohansemmanuel.com/react-hooks-documentation-easy-to-read/) is another article about hooks, which provides a simpler version of the official React hooks documentation.
 
+[Watch this video](https://youtu.be/-MlNBTSg_Ww) for a look into useState(), useEffect() and custom hooks in great detail.
```

Comment:
Can you surround the method names in back-ticks.

Also, can you bullet point the additional resources, they should not have a blank line between them.

Reply:
done